--- openmotif-2.2.2/demos/programs/animate/Makefile.am.orig	Sun Feb  2 22:16:01 2003
+++ openmotif-2.2.2/demos/programs/animate/Makefile.am	Sun Feb  2 22:16:19 2003
@@ -5,7 +5,7 @@
     superman5.xbm superman6.xbm Imakefile XmdAnimate.ad \
     README animate.help animate.man xmanimate.man
 
-animate_SOURCES = xmanimate.c animate.c
+animate_SOURCES = xmanimate.c
 
 CLEANFILES =  dog.uid plane.uid superman.uid xmanimate.uid
 
