--- openmotif-2.2.2/clients/uil/Makefile.am	Wed Feb  6 19:39:16 2002
+++ openmotif-2.2.2.new/clients/uil/Makefile.am	Sun Feb  2 19:55:43 2003
@@ -11,7 +11,7 @@
 
 libUil_la_LIBADD = UilParser.lo ../../lib/Mrm/libMrm.la ../../lib/Xm/libXm.la 
 
-uil_LDADD = ../../lib/Mrm/libMrm.la ../../lib/Xm/libXm.la 
+uil_LDADD = ../../lib/Mrm/libMrm.la ../../lib/Xm/libXm.la libUil.la
 
 INCLUDES = -DINCDIR=\"@INCDIR@\" \
            -DLIBDIR=\"@LIBDIR@\" \
@@ -67,7 +67,7 @@
 UilDBDef.h:
 	$(LN_S) $(srcdir)/../../tools/wml/UilDBDef.h UilDBDef.h
 
-uil_SOURCES = $(SRCS2)
+uil_SOURCES = UilMain.c
 
 if MessageCatalog
 
